pkg/client/unversioned/cache/poller.go

```diff
@@ -1,86 +0,0 @@
-/*
```

Comment:
The poller is not used anywhere in the code.
However, it uses Replace method, which I would need to update, so I decided to just remove it.